### PR TITLE
Cleaning the scripts we use to manage the contracts owned by the multisig

### PR DESCRIPTION
**Pull request by juniset**

### CODE REVIEW

File: scripts/execute_script.sh [hunk 1]
```diff
@@ -6,7 +6,10 @@
 
 set -e # stop the script if any subprocess fails
 
-etherlime compile --runs 999
+# rm -rf build
+# npm run compile:lib
+# npm run compile
+# npm run compile:legacy
```
Comment (elenadimitrova) [inline scripts/execute_script.sh:12]: I'm for this change considering how long compilation takes currently however the obvious risk is that we execute scripts without compiling to latest version of the contracts. As a prevention, can we add some warning text in the comments header of scripts using `execute_script` to say `COMPILE CONTRACTS BEFORE RUNNING` or something similar?

File: scripts/update_compound_registry.js [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+// Script to add/remove CTokens on Compound
+//
+// To add a token:
+// ./execute_script.sh update_compound_registry.js <network> --add --token <token address> --ctoken <ctoken address>
+//
+// To remove a token:
+// ./execute_script.sh update_compound_registry.js <network> --remove --token <token address>
+//
+// where:
+//    - network = [test, staging, prod]
```
Comment (elenadimitrova) [inline scripts/update_compound_registry.js:11]: Can the network not be `ganache` as well for local testing?

File: scripts/update_wallet_in_factory.js [hunk 3]
```diff
@@ -1,4 +1,12 @@
-// Usage: ./execute.sh updateBaseWallet.js staging
+// ///////////////////////////////////////////////////////////////////
+// Script to update the BaseWalet implementation in the WalletFactory contract
+//
+// Can be executed as:
+// ./execute_script.sh update_wallet_in_factory.js <network>
+//
+// where:
+//     - network = [test, staging, prod]
```
Comment (elenadimitrova) [inline scripts/update_wallet_in_factory.js:8]: Can we remove `defaultNetwork = "test"`  in this script to mandate passing the network property

File: scripts/execute_script.sh (hunk 1 above)
Comment (juniset) [reply scripts/execute_script.sh:12]: Agreed. Actually I've added a `--no-compile` flag in #156 (https://github.com/argentlabs/argent-contracts/pull/156/files#diff-0aa9ab67bc9291891ca0b035f24dce95R12) to address that. We can either wait for #156 to be merged or I can include that change in this PR.

Comment (elenadimitrova) [reply scripts/execute_script.sh:12]: I don't mind either way.

**Review by elenadimitrova — APPROVED** [review]

**Pull request merged**